config/config.go

```diff
@@ -306,7 +306,10 @@ func ParseConfig() (*Config, error) {
 	if err != nil {
 		return nil, err
 	}
-
+	conf.HsmConfig, err = ParseHSMConfig()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
If the hsm config file doesn't exist we should use the HSM config from loom.yml, so this should look something like:
```golang
hsmCfg, err = ParseHSMConfig()
if err == nil {
  conf.HsmConfig = hsmCfg
} else if _, notFound := err.(viper.ConfigFileNotFoundError); !notFound {
  // loom_hsm config file exists but couldn't be loaded
  return nil, errors.Wrap(err, "failed to load loom_hsm config")
}
```